test/jasmine/tests/config_test.js

```diff
@@ -594,6 +601,13 @@ describe('config argument', function() {
             .then(delay(RESIZE_DELAY))
             .then(function() {
                 checkLayoutSize(elWidth / 2, elHeight / 2);
+
+                var mainSvgs = document.getElementsByClassName('main-svg');
+                checkElementsSize(mainSvgs, elWidth / 2, elHeight / 2);
+
+                var canvases = document.getElementsByTagName('canvas');
```

Comment:
Checking the size of the `canvas` elements sandwiched between the SVG elements is also very important.